pyds8k/base.py

```diff
@@ -128,7 +128,8 @@ def one(self, route, resource_id, rebuild_url=False):
             get_resource_by_route = \
                 import_module('{}.resources.utils'.format(__package__)
                               ).get_resource_by_route
-        url = self._set_url(route, resource_id, rebuild_url=rebuild_url)
+        url = self._set_url(route.split('.')[-1],
```

Comment:
I'd like route.split('.')[-1] in _set_url, or
cs/pprc base_url without 'cs', and then in _set_url: route = route.replace('.', '/')
so the variable route is actually a route note a resource_name, and base_url is really 'base'

Reply:
Either is ok for me.